test/unittests/mongoBackend/mongoGetSubscriptions_test.cpp

```diff
@@ -96,12 +96,12 @@ static void prepareDatabaseV1Subs(void) {
                                                        "type" << "ONCHANGE" <<
                                                        "value" << BSON_ARRAY("AX2" << "AY2")
                                                        )) <<
-                        "throttling" << 5.0
+                        "throttling" << (long long) 5
                         );
 
     BSONObj sub3 = BSON("_id" << OID(SUB_OID3) <<
-                        "expiration" << 20000000 <<
-                        "lastNotification" << 25000000 <<
+                        "expiration" << (long long) 20000000 <<
```

Comment:
These changes here of expiration, count and throttling I don't like too much.
I tried to remove the **(long long)**  for these fields but everything went south, so finally I rollbacked that change.

We will take care of all this in issue #1455